.bazelci/postsubmit.yml

```diff
@@ -1,22 +1,67 @@
 ---
 platforms:
   ubuntu1604:
+    bazel: last_green
     build_targets:
-    - "//..."
-    # test_targets:
-    # - "//..."
+    - "//src/..."
+    - "//test/..."
+    - "//android/..."
+    - "//rules/..."
+    - "-//src/java/com/example/sampleapp/..."
+    - "-//src/tools/ak/..."
+    - "//toolchains/..."
+    - "//tools/..."
+    test_targets:
+    - "//src/..."
+    - "//test/..."
+    - "-//src/tools/..."
+    - "-//src/java/com/example/sampleapp/..."
   ubuntu1804:
+    bazel: last_green
     build_targets:
-    - "//..."
-    # test_targets:
-    # - "//..."
+    - "//src/..."
```

Comment:
All the targets are included except //examples/..., which need to be build in a separate task, changing the directroy where their WORKSPACE file is.

The examples build works with Bazel@HEAD and using 2 extra flags:
`bazel build --experimental_google_legacy_api --experimental_enable_android_migration_apis //...`

Reply:
Yeah, I think we added these to the repo-level .bazelrc already.